### ISSUE TITLE
macOS build?

### ISSUE THREAD

**Issue by pavel-shpilev** [nanopool/Claymore-Dual-Miner#95]
Hi. I mine on my MBP with AMD 460 and a 1080Ti eGPU. I have successfully compiled ETH, XMR, ZEC and some other miners under macOS. I even get slightly better hash rates in mac builds than in Bootcamp Windows for the same miners.
I would love to be able to run Claymore without the need to boot to Windows. Any chance of mac binaries?
Thank you

**Comment by ttrentler** [comment]
Curious...  how man eGPUS can your Mac support?

**Comment by 1mike12** [comment]
a macbook pro 2016 15" has 4 thunderbolt slots, I'm not sure about what the macos will do, but i know in windows you should be able to use at least 2. https://www.reddit.com/r/eGPU/comments/6u9lg6/multiple_egpu_on_15_macbook_pro_2017_possible/

**Comment by j4ys0n** [comment]
I'd love to run the claymore miner on my mac. can i download the source and compile it? i can't seem to find the source anywhere...

ethminer runs great, hash speed is better than on linux & windows.

**Comment by Humanovation** [comment]
I'm running Ethminer on my iMac Pro using a Sonnet 550/Radeon RX580 eGPU and am only getting 4.25 Mh/s via ethermine.org.  Seems like I should be a lot faster than that.  Are your numbers similar?

**Comment by 1mike12** [comment]
@Humanovation that's about right. That figure is probably not even from your rx580. I have the same exact setup and I think it's actually mining from the internal gpu instead.

**Comment by Humanovation** [comment]
> @Humanovation that's about right. That figure is probably not even from your rx580. I have the same exact setup and I think it's actually mining from the internal gpu instead.

No, that is what I get from the eGPU 580 in OSX ... I get about 12 MH/s from the internal Vega 56:
 m  15:08:41|          Speed  11.82 Mh/s    gpu/0  7.51  gpu/1  4.31  [A0+0:R0+0:F0] Time: 00:01

**Comment by pminer** [comment]
Hello im mining ethereum with a ASUS ROG STRIX RX570 OC 4GB and im getting 28Mhs with claymore.
I can mod any bios to 28Mhs minimun, with minimal energy consumption, which means less heating and longer life, if you need help send me a message.

**Comment by pminer** [comment]
GPU´s Bios Mods for mining Ethereum

List of GPU´s any brand or RAM
R9 290,
R9 290X,
R9 390,
R9 390X,
RX 470
RX 480
RX 570
RX 580

On all bios we guarantee 25Mhs minimum and more energy efficiency, which means less heating and longer life.

Any question, contact me.

**Comment by 1mike12** [comment]
@pminer fuck off

**Comment by rgaufman** [comment]
Anyone managed to get this working on a mac? :) - I have a hackintosh with a couple GPUs and it's a pain to re boot to Linux/Windows to mine, lol
